### PR TITLE
feature: customer call returns session token for guest users

### PR DESCRIPTION
**Pull request by johnforte**
### Your checklist for this pull request
Thanks for sending a pull request! Please make sure you click the link above to view the contribution guidelines, then fill out the blanks below.

🚨Please review the [guidelines for contributing](./CONTRIBUTING.md) to this repository.

- [X] Make sure you are making a pull request against the **develop branch** (left side). Also you should start *your branch* off *our develop*.
- [X] Make sure you are requesting to pull request from a **topic/feature/bugfix/devops branch** (right side). Don't pull request from your master!
- [ ] Have you ensured/updated that CLI tests to extend coverage to any new logic. Learn how to modify the tests [here](https://woographql.com/contributing/2-local-testing).

What does this implement/fix? Explain your changes.
---------------------------------------------------
Since the woocommerce session takes into account that a guest user session can generate a token, this should be exposable thru the customer sessionToken trait. 

Does this close any currently open issues?
------------------------------------------
No.

Any relevant logs, error output, GraphiQL screenshots, etc?
-------------------------------------
(If it’s long, please paste to https://ghostbin.com/ and insert the link here.)

Any other comments?
-------------------
…

Where has this been tested?
---------------------------

- **WooGraphQL Version:** 0.10.3 
- **WPGraphQL Version:** 1.6.3
- **WordPress Version:** 5.8
- **WooCommerce Version:** 5.5.2

### CODE REVIEW

File: includes/type/object/class-customer-type.php [hunk 1]
```diff
@@ -115,11 +115,7 @@ public static function register() {
 						'type'        => 'String',
 						'description' => __( 'A JWT token that can be used in future requests to for WooCommerce session identification', 'wp-graphql-woocommerce' ),
 						'resolve'     => function( $source ) {
-							if ( \get_current_user_id() === $source->ID ) {
 								return apply_filters( 'graphql_customer_session_token', \WC()->session->build_token() );
-							}
-
-							return null;
```
Comment (kidunot89) [inline includes/type/object/class-customer-type.php:122]: Why not simply change the conditional to evaluate whether the `$source->ID` is a WooCommerce guest user ID. They tend to be random string. 🤔

**Review by kidunot89 — CHANGES_REQUESTED** [review]
@johnforte, I understand what you are trying to solve here, but this would remove all context for this field resulting in admin getting there own JWTs exposed.

File: includes/type/object/class-customer-type.php (hunk 1 above)
Comment (johnforte) [reply includes/type/object/class-customer-type.php:122]: ok I will take a look into it, right now the 
```
$source->ID
```
does not contain a value if the user is a guest. From a quick look at the extending type for customers, it maybe just be a scoping issue. I have to dig a bit deeper to confirm that or see why the id of a guest user isn't populating in the source variable.

**Review by kidunot89 — APPROVED** [review]
@johnforte Perfect. Thanks for the work on this 🙏🏿

**Pull request merged**